### PR TITLE
Add Red Hat Project Initializer

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### What does this PR do?

Add Red Hat Project Initializer VSCode extension to registry

### CODE REVIEW

File: plugins/redhat.project-initializer/0.0.9/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+type: VS Code extension
+name: Project Initializer
+title: Project Initializer
+description: A lightweight extension based on Red Hat launcher to generate quickstart projects using Eclipse Che
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/project-initializer/0.0.9/vspackage
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: Red Hat, Inc.
+repository: https://github.com/fabric8-launcher/launcher-vscode-extension
+category: Other
+firstPublicationDate: "2019-04-18"
```
Comment (ibuziuk) [inline plugins/redhat.project-initializer/0.0.9/meta.yaml]: No new line

File: plugins/redhat.project-initializer/0.0.9/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+id: redhat.project-initializer
+version: 0.0.9
+type: VS Code extension
+name: Project Initializer
+title: Project Initializer
+description: A lightweight extension based on Red Hat launcher to generate quickstart projects using Eclipse Che
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/project-initializer/0.0.9/vspackage
```
Comment (ibuziuk) [inline plugins/redhat.project-initializer/0.0.9/meta.yaml]: please use vsix from GH release to avoid marketplace rate limit error

**Comment by garagatyi** [comment]
Please note that I added CircleCI and Eclipse sign off check to required checks for PRs.
Since this PR was created before CircleCI was configured in the repository you have to commit something to this PR to initiate CircleCI build that is required to proceed with merging the PR.
Sorry for the inconvenience.

**Comment by apupier** [comment]
> Please note that I added CircleCI and Eclipse sign off check to required checks for PRs.
> Since this PR was created before CircleCI was configured in the repository you have to commit something to this PR to initiate CircleCI build that is required to proceed with merging the PR.
> Sorry for the inconvenience.

it is required to rebase on master so no need to have a dummy commit

it might be good to provide the v3 (and v2?) version of the description in this PR directly.

**Comment by apupier** [comment]
created PR for v2 and v3 https://github.com/eclipse/che-plugin-registry/pull/141

**Comment by jeffmaury** [comment]
Replaced by #141

**Pull request closed without merging**